Makefile.am

```diff
@@ -134,6 +134,13 @@ test_run_tests_CFLAGS =
 else
 test_run_tests_CFLAGS = -Wno-long-long
 endif
+
+if SUNOS
+# Can't be turned into a CC_CHECK_CFLAGS in configure.ac, it makes compilers
```

Comment:
Sounds good. Should we enable `-pthread` on platforms that are not Solaris, Android and z/OS` when building with autotools too so that it's consistent with builds done with gyp?